app/models/expense.rb

```diff
@@ -1,3 +1,10 @@
 class Expense < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  belongs_to :category
+  has_many :categories_expenses, dependent: :destroy
+  has_many :categories, through: :categories_expenses
+
+  validates :name, presence: true
+  validates :amount, presence: true
```

Comment:
- [ ] Great progress! However, there's a key improvement to consider. As mentioned by the previous reviewer, it's crucial to implement validations to prevent users from entering excessively large amounts or unexpected input. This ensures that the page remains responsive and avoids potential issues with design elements. 


![image](https://github.com/evansnyamekye/budget-app/assets/55840999/54be743a-9e47-4593-8b01-d59a31fbaac5)

- [ ] It is important to include error messages for instances where users enter alphanumeric characters in the number or amount fields to provide a more seamless user experience. Currently, if a user enters alphanumeric characters, the input is being changed to zero. However, it would be better to display an error message to the user and allow them to input a numeric value instead.


https://github.com/evansnyamekye/budget-app/assets/55840999/785af372-fd59-45f3-82d9-695f53590bfb


Keep up the good work, and feel free to reach out if you have any questions or need further assistance! 🚀🛠️